plugins/modules/efs.py

```diff
@@ -72,7 +72,7 @@
     throughput_mode:
         description:
             - The throughput_mode for the file system to be created.
```

Comment:
Can you please add a sentence like "Support for C(elastic) was added in version 7.1.0."